CONTRIBUTING.md

```diff
@@ -5,48 +5,84 @@ Want to contribute to Sitecore JavaScript Services? There are a few things you n
 ## Pre-requisites:
 
 - `node.js` ([Active LTS](https://nodejs.org/en/about/releases/) version) installed (cmd `node -v` to test).
-- `npm` (6.x or later) installed (cmd `npm -v` to test).
+- `npm` (6.x) installed (cmd `npm -v` to test).
+
+## Developing
+
+See Branching overview below - We use `dev` for our current development.
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) this repository to your own GitHub account and then [clone](https://help.github.com/articles/cloning-a-repository/) it to your local device.
+2. Create a new branch e.g. `git switch -c feature/my-jss-feature`
+3. When you're happy with your changes, open a Pull Request targeting the `dev` branch of the `sitecore/jss` repository. 
+4. Note: CI will run lint for all packages and samples, as well as tests for all packages. Please make sure these pass or your PR can not be merged.
```

Comment:
We can add in another point that they must add the dev reviewers group to the reviewer's list after making changes to the PR.

Reply:
I have this in there, maybe it's not pushed yet. Stand by :)